tests/integrationtest/t/ddl/attributes_sql.test

```diff
@@ -0,0 +1,44 @@
+# TestAlterTableAttributes
+create table alter_t (c int);
+alter table alter_t attributes="merge_option=allow";
+alter table alter_t attributes="merge_option=allow,key=value";
+alter table alter_t attributes=" merge_option=allow ";
+alter table alter_t attributes=" merge_option = allow , key = value ";
+alter table alter_t attributes " merge_option=allow ";
+alter table alter_t attributes " merge_option=allow , key=value ";
+drop table alter_t;
+
+# TestDropSchema
+create database ddl__attributes_sql_test;
+use ddl__attributes_sql_test;
+create table drop_s1 (c int)
+PARTITION BY RANGE (c) (
+	PARTITION p0 VALUES LESS THAN (6),
+	PARTITION p1 VALUES LESS THAN (11)
+);
+create table drop_s2 (c int);
+alter table drop_s1 attributes="key=value";
+alter table drop_s1 partition p0 attributes="key1=value1";
+alter table drop_s2 attributes="key=value";
+select count(*) from information_schema.attributes where ID like 'schema/ddl__attributes_sql_test%';
+drop database ddl__attributes_sql_test;
+select count(*) from information_schema.attributes where ID like 'schema/ddl__attributes_sql_test%';
+
+# TestDefaultKeyword
+use ddl__attributes_sql;
```

Comment:
Is this necessary?

Reply:
The previous test used another database called `ddl__attributes_sql_test`, so we need run `use ddl__attributes_sql` here.

But I think it's better move it into previous test case. So I updated it.

Reply:
ok